tests/integrations/coprocessor/test_select.rs

```diff
@@ -76,6 +77,7 @@ fn test_select() {
         let result_encoded = datum::encode_value(&mut EvalContext::default(), &row).unwrap();
         assert_eq!(result_encoded, &*expected_encoded);
     }
+    assert!(limiter.total_read_bytes_consumed(true) > 0); // the consume_sample is called due to read bytes quota
```

Comment:
why not give a solid number?

Reply:
Here the logic is to verify if the read bytes based limiter is called. We don't know the actual response size here and so no way to get accurate wait time. 
For actual accurate wait time, it's covered by unit test.